doc/source/api/visualization/plotter_windows_manager.rst

```diff
@@ -7,43 +7,47 @@ The ``PlotterWindowsManager`` class provides for managing and directly interacti
 with Matplotlib windows. By registering these methods with ``EventsManager``, you can
 update plots during run time.
 
-The following example updates ``window-1`` and ``window-2`` during solution initialization 
+The following example updates ``p_xy`` and ``p_res`` during solution initialization
```

Comment:
@prmukherj This statement leads me to anticipate that "solution initialization" is something that's done in the example script, but it is not.  The statement "it also updates both windows at the end of every time step" makes that seem incidental. All of updates described happen because and only because of the explicit callback registrations. It's important to mention that.